src/component/homepage/Home.jsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import Card from './Card'
+import axios from 'axios'
 function Home() {
+  const [article, Setarticle] = useState([])
```

Comment:
The set function should not start with a capital letter, this is not a component or a class.